podman_compose.py

```diff
@@ -833,6 +833,12 @@ def get_network_create_args(net_desc, proj_name, net_name):
     ipam_config_ls = ipam.get("config", [])
     if net_desc.get("enable_ipv6"):
         args.append("--ipv6")
+    if net_desc.get("ignore"):
+        args.append("--ignore")
+    if net_desc.get("disable_dns"):
```

Comment:
Should be called `x-podman.disable_dns`.

Reply:
Fixed.